style.css

```diff
@@ -951,226 +868,159 @@ input {
     margin-bottom: 0;
   }
 
-  .main__product {
+  .product {
     margin-bottom: 70px;
   }
 
-  .main__product_item:nth-child(n+2) {
+  .product__content:nth-child(n+2) {
     display: none;
     max-width: none;
   }
 
-  .main__product_line {
+  .product__line {
     display: none;
     max-width: none;
   }
-  
-  .main__product_item {
+
+  .product__content {
     flex-basis: 200px;
   }
-  
-  .main__product_item img {
+
+  .product___img {
     width: 80%;
     display: inline-block;
   }
 
-  .main__product_name {
+  .product__name {
     font-size: 15px;
-    line-height: 14px;  
+    line-height: 14px;
   }
 
-  .main__product_price {
-    font: bolder 20px/19px montserratregular;
+  .product__price {
+    font: bolder 20px/19px Gotham Pro, sans-serif;
   }
 
-  .main__product_silver-circle {
-    width: 15px;
-    height: 15px;
-    -moz-border-radius: 50%;
-    -webkit-border-radius: 50%;
-    border-radius: 50%;
-    background: #C4C4C4;
-    margin-right: 5px;
-  } 
-
-  .main__product_black-cirle {
-    width: 15px;
-    height: 15px;
-    -moz-border-radius: 50%;
-    -webkit-border-radius: 50%;
-    border-radius: 50%;
-    background: #0E1211;
-    margin-right: 5px;
-  } 
-
-  .main__product_red-circle {
-    width: 15px;
+  .product_item {
+  	width: 15px;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 4 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Mandri_Firsttask/pullRequest?prid=5484013)